### PR TITLE
Prepare release 0.1.0

### PR DESCRIPTION
**Pull request by w-vi**
Includes automatic deploy from travis on tag push

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,16 +2,25 @@ sudo: required
...
 script:
-  - python -B -m unittest discover
-  - pip install -e .
-  - bundle exec cucumber
+- python -B -m unittest discover
+- pip install -e .
+- bundle exec cucumber
+deploy:
+  provider: pypi
+  user: wvi
```
Comment (kylef) [inline .travis.yml]: Can we use a shared Apiary account? Similar to `apiary-sre` account for NPM.

File: .travis.yml [hunk 2]
```diff
@@ -2,16 +2,25 @@ sudo: required
...
+  password:
+    secure: G2gtzJE0zn/AnS7Z7RPiyTJMV583yJitBg8Un4/yA1mr9eNzEDw7s8EA1CRYMN/y9rizQ/xbtqonqTSjWdXJOu4g0+tXMUC3CZ5BqMxWVaQBQGx7XRF9t0YolhzznMo4i9rtwVAo8Bq0KlOWtjFpbTW2G0PqvIGWgUv/xo/JabWj6YyDmtnciF1KxtfVlD/RoSnCztyOzpqQqr6nQPLK221acELYCAAIE13L/3tJm8ChpGsDW8My3Z3gxQd50rQW7Z79CPxQci3reTTCx45uOOPLqT4oBUH6w1WXy9Aj/yjJAMchlQBS9i1MdAkqCClNlgpdvX928rQy6PkGGTxB/TlcgIXToMr7N/AlfqohVLrkE4iNHi51H3m9+PROzctfrpl3L4pos55ogG9gjeLPOMQaq8w7khHyEcyJMOEfrfuQYYs/enavq7fdFM8Ia92gCwowjLWaYXSsHm4t614a+as0gjmV96NOi9ClFqMDVIgwI8PPFoKuAKwmuus017heTphB9elAeIPWVQGHiF3vz/DorG0Kf6mo8Hq1mJMuO3pFgbEP5PH8e2LBwYofbv7GPdqMgDZ3pzUZeAVZEfKXmATA2JtvutqGJEPvfd79h/idhHQcaXK+W/pzShLck+EOPG9gzcWccpNmlioQ09YvOLyN6CDDF/jz0qDvinu0tHM=
+  on:
```
Comment (kylef) [inline .travis.yml]: Not fully sure I understand how this works, but from what I'm making out from the Travis CI documentation, these steps are run for each Python version above. Therefore we might want to limit it to a certain version, i.e, via:

```yaml
on:
  python: '3.5'
```

https://docs.travis-ci.com/user/deployment#Conditional-Releases-with-on%3A

> For language runtimes that support multiple versions, you can limit the deployment to happen only on the job that matches the desired version.

File: .travis.yml [hunk 3]
```diff
@@ -2,16 +2,25 @@ sudo: required
...
+    tags: true
+    distributions: sdist bdist_wheel
```
Comment (kylef) [inline .travis.yml]: Distributions shouldn't be inside `on` section but inside the `deploy` section.

Ref: https://docs.travis-ci.com/user/deployment/pypi#Uploading-different-distributions

File: .travis.yml (hunk 1 above)
Comment (w-vi) [inline .travis.yml]: There is none at the moment, we should create one but I haven't persuaded sre yet.

File: .travis.yml (hunk 3 above)
Comment (w-vi) [inline .travis.yml]: this was generated by travis cli tool not written by hand so I trust them but maybe it is incorrect, let me check it one more time.

Comment (kylef) [inline .travis.yml]: Seems to be a known bug on Travis CI tool: https://github.com/travis-ci/travis.rb/issues/373

**Comment by w-vi** [comment]
Fixed according to spec, the travis cli tool is almost useless in this case :/

**Comment by kylef** [comment]
👍

File: .travis.yml [hunk 4]
```diff
@@ -2,16 +2,26 @@ sudo: required
...
+- '3.4'
+- '3.5'
 script:
-  - python -B -m unittest discover
-  - pip install -e .
-  - bundle exec cucumber
+- python -B -m unittest discover
+- pip install -e .
+- bundle exec cucumber
+deploy:
```
Comment (honzajavorek) [inline .travis.yml]: 👍

**Pull request merged**